roles/template_service_broker/tasks/remove.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
What do we do when any of these commands fail?  How do we ensure that these objects are removed?  Just something to think about.